src/views/FileBrowserView/PublishFileBrowser.vue

```diff
@@ -278,6 +291,31 @@ export default {
       return publishRest.assetDownloadURI(this.identifier, this.version, asset_id);
     },
 
+    viewURI(asset_id, name) {
+      const file_ext = this.fileExtension(name);
+      if (file_ext === 'nii') {
+        return `https://bioimagesuiteweb.github.io/alphaapp/viewer.html?image=${publishRest.assetDownloadURI(this.identifier, this.version, asset_id)}`;
+      }
+      if (file_ext === 'nwb') {
+        return `http://nwbexplorer.opensourcebrain.org/nwbfile=${publishRest.assetDownloadURI(this.identifier, this.version, asset_id)}`;
+      }
+
+      return undefined;
+    },
+
+    fileExtension(name) {
+      let ext;
+      const name_split_ar = name.split('.');
+      if (name_split_ar[name_split_ar.length - 1] === 'nwb') {
+        ext = 'nwb';
+      } else if (name_split_ar[name_split_ar.length - 1] === 'nii' || name_split_ar[name_split_ar.length - 2] === 'nii') {
+        ext = 'nii';
+      } else {
+        ext = false;
```

Comment:
While returning either a `boolean` or `string` based on the function input is a completely "legal" operation in JavaScript, I would advise against it.
1) It's not great for code readability, especially when there's places in the code that rely on this side effect. 
2) We have been moving a lot of the Vue code to TypeScript (where this is not valid), so this will eventually need to be refactored anyways.

I don't think it's necessary to parse the file extension though, and I agree with @yarikoptic's suggestion at https://github.com/datalad/datalad-deprecated/blob/master/datalad_deprecated/resources/website/assets/js/main.js#L21. It'd be really nice to have a single place where you can simply add new file extensions and regexes and have things just work.